aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -254,14 +266,10 @@ module aptos_framework::aptos_governance {
 
     /// Initializes the state for Aptos Governance partial voting. Can only be called through Aptos governance
     /// proposals with a signer for the aptos_framework (0x1) account.
-    public fun initialize_partial_voting(
-        aptos_framework: &signer,
-    ) {
+    public fun initialize_partial_voting(aptos_framework: &signer,) {
         system_addresses::assert_aptos_framework(aptos_framework);
 
-        move_to(aptos_framework, VotingRecordsV2 {
-            votes: smart_table::new(),
-        });
+        move_to(aptos_framework, VotingRecordsV2 { votes: smart_table::new(), });
```

Comment:
Can the formatter remove the trailing , as well when it's one liner like this?

Reply:
Yeah, I agree with you